recipe/meta.yaml

```diff
@@ -1,16 +1,17 @@
 {% set name = "CyRK" %}
-{% set version = "0.12.2" %}
+{% set version = "0.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/cyrk-{{ version }}.tar.gz
-  sha256: 8d9eabf65509660a4785bd861ceb479edc877a04b8a39aea8c0faffae0fbbc6e
+  sha256: 3e57e33522d00b518ed4f7ab290d2827d2310c5f91af6c1a15d0c42d757c617f
 
 build:
-  skip: true  # [py>312]
+  skip: true  # [py>=314]
```

Comment:
No need to worry about this since Python version support is handled by bots issuing migration PRs.